pkgs/misc/drivers/utsushi/patches/series

```diff
@@ -0,0 +1,3 @@
+absolute-path-to-convert.patch
+print-errors.patch
+absolute_path_for_tesseract.patch
```

Comment:
If the patches have to be modified in the future (there *are* updates of this driver, the last one was this year) it will be easier to do this with quilt and this file. But I agree this is a very small advantage.